pkgs/by-name/co/comictagger/package.nix

```
@@ -0,0 +1,92 @@
+{
+  fetchPypi,
+  imagemagick,
+  lib,
+  nix-update-script,
+  python3Packages,
+  qt5,
+  stdenvNoCC,
+  versionCheckHook,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "comictagger";
+  version = "1.5.5";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-f/SS6mo5zIcNBN/FRMhRPMNOeB1BIqBhsAogjsmdjB0=";
+  };
+
+  nativeBuildInputs = [
+    qt5.wrapQtAppsHook
+    versionCheckHook
```

```suggestion
    versionCheckHook
    imagemagick
```
This is the preferred way of using tools at build time.

---

Awesome, thanks!